.github/workflows/dco-check.yml

```diff
@@ -0,0 +1,36 @@
+name: DCO Check
+
+on:
```

Comment:
Please update CONTRIBUTING.md with information about this check. So that new contributors knows exactly what is expected.